user_input.py

```diff
@@ -0,0 +1,59 @@
+from textwrap import dedent
+from time import sleep
+from random import choice
+
+savage1 = {"ducky": "Has feathers and goes quack quack", "red": "The colour your face should be", "cup": "sippy sippy", "feeble": "An assessment of your mental prowess..."}
+
+
+def check_input(user_input):
+
+    evaluation = None
+
+    try:
+        int(user_input)
+
+    except ValueError:
+        sleep(2)
+        print("User, you were given precise instructions.")
+        sleep(2)
+        print("Integers 1 - 4 only.")
+        sleep(2)
+        print("Let's try again shall we...")
+        sleep(2)
+        return evaluation
+    
+    except:
+        sleep(2)
+        print("Ah... trying to be clever I see...")
+        sleep(2)
+    
+    user_input = int(user_input)
+
+    if (user_input) not in range(1, 5):
+        sleep(2)
+        print("Regardless of your verbal intellect player, you seem to be struggling with basic maths.")
+        sleep(2)
+        print("Let's try again shall we...")
+        sleep(3)
+
+    else:
+        evaluation = True
+    
+    return evaluation
+
+
+
+def check_response(wrong, conc_wrong):
```

Comment:
The name of the function and its behaviour live in different conceptual domains!

The name `check_response` implies some form of validation and returning a boolean / object extending boolean concepts. However the behaviour is to return `None` - ie. not to be used for validation - and to interact with the user.

This makes reading and debugging and extending code harder and more confusing.